src/js/tech/tech.js

```diff
@@ -218,12 +218,13 @@ class Tech extends Component {
       return;
     }
 
+    let player = this.player();
     let textTracksChanges = function() {
       let updateDisplay = Fn.bind(this, function() {
-        this.trigger('texttrackchange');
+        player.trigger('texttrackchange');
```

Comment:
Not sure. I guess this isn't quite ready. I only looked quickly at this and this was a quick fix for the issue. It worked locally.
Ill take another look later.